src/main/java/it/pagopa/pn/delivery/svc/NotificationReceiverValidator.java

```diff
@@ -26,6 +28,7 @@ public class NotificationReceiverValidator {
     private final MVPParameterConsumer mvpParameterConsumer;
     private final ValidateUtils validateUtils;
     private final PnDeliveryConfigs pnDeliveryConfigs;
+    private static final String PHYSICAL_ADDRESS_PATTERN = "[0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ./' -]+";
```

Comment:
Riusciamo a portare l'elenco dei caratteri supportati in configurazione ed iniettarli nel codice? 
Quest'elenco possiamo lasciarlo come default